src/iss.js

```diff
@@ -151,16 +152,46 @@ export async function request(
     return JSON.parse(response.text);
 }
 
+/*
+ * Loads the transportTime property from google
+ * and adds it to the given Service instances
+ */
+async function attachTransportTimes(
+    services: Array<Service>
+): Promise<Array<Service>> {
+    let formatPoint = (point) => `${point.lat},${point.lon}`;
+    const maps = await Maps();
+    let service: ?Service;
+    let travelTimes = await maps.travelTime(
+        [/*::`*/
+        for (service of services)
+            if (!service.Location().isConfidential())
+                formatPoint(service.location.point)
+        /*::`*/]
```

Comment:
Still has weird indenting. I expect the three rows to be flat. Consider repatriating `[]` with `()`